dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.org.eclipse.microprofile.rest.client-1.4.feature

```diff
@@ -5,7 +5,8 @@ singleton=true
 -features=com.ibm.websphere.appserver.javax.cdi-2.0, \
  com.ibm.websphere.appserver.javax.annotation-1.3, \
  com.ibm.websphere.appserver.javax.jaxrs-2.1, \
- com.ibm.websphere.appserver.org.eclipse.microprofile.config-1.3; ibm.tolerates:=1.4
+ com.ibm.websphere.appserver.org.eclipse.microprofile.config-1.3; ibm.tolerates:=1.4,\
+ io.openliberty.mpCompatible-0.0
```

Comment:
Why was the following -bundles not deleted, as you have done for MP Health? Any difference between Rest Client and MP Health?

Reply:
You're right, the MP Rest Client API should be a private feature too.

Reply:
Since the PB for this PR was green, probably best to change the Rest Client API in a follow-up PR.

Reply:
This _is_ the private API feature